test/common/src/main/kotlin/org/codice/compliance/verification/binding/PostBindingVerifier.kt

```diff
@@ -36,10 +40,44 @@ class PostBindingVerifier(private val response: IdpPostResponseDecorator) {
      * Verify the response for a post binding
      */
     fun verify() {
+        verifyNoNulls()
         decodeAndVerify()
-        verifySsoPost()
+        verifyPostSSO()
```

Comment:
This is outside the scope of this PR, but I think we should move this method call out to the tests themselves. Because this binding verifier should be profile agnostic. Does anyone disagree?